osg-test-log-viewer

```diff
@@ -35,7 +37,8 @@ def load_logdata_from_handle(handle):
     current_test = ""
 
     for line in handle:
-        line = line.rstrip()
+        if six.PY3:
+            line = line.decode(errors="replace").rstrip()
```

Comment:
I presume `six.PY3` checks whether python3 is running - but what happens if this is false?  (Does `.rstrip()` still get run if it's python2?)

Reply:
Derp. Fixed.

Reply:
Yay, the system works! :D